devel/hello-world.lua

```diff
@@ -32,6 +42,30 @@ function HelloWorldWindow:init()
             frame={w=10, h=5},
             frame_style=gui.INTERIOR_FRAME,
         },
+        widgets.Divider{
+            frame={l=0,t=3}
+        },
+        widgets.CycleHotkeyLabel{
+            view_id='level',
+            frame={l=0, t=3, w=16},
```

Comment:
setting `t` properties in frames is not useful for autoarranged widgets

Reply:
however, since 2 lines is way too many to put between the CycleHotkeyLabel and the Slider widget, maybe create a containing Panel that does not have autolayout properties.